### ISSUE TITLE
are reference sites physically the same as recording sites?

### ISSUE THREAD

**Issue by agbondy** [cortex-lab/neuropixels#29]
In the current version of Neuropixels (1.0), are the three reference sites physically the same as the recording sites? And if so, can you use the reference sites just like normal recording channels (so long as you don't reference anything to them obviously).

**Comment by nsteinme** [comment]
Yes they are physically the same, but no, you can't use them like normal
recording channels.
